FoursquareTopTests/VenueViewModel+Builder.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  VenueViewModel+Builder.swift
+//  FoursquareTop
+//
+//  Created by Fran on 9/21/16.
+//  Copyright © 2016 Karumi. All rights reserved.
+
+
+import Foundation
+import MapKit
+
+@testable import FoursquareTop
+
+extension VenueViewModel {
```

Comment:
So @davideme @gragera shall we stick to the builder extension in order to keep consistency  across our project's code?